x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/dashboard_sections/benchmarks_section.tsx

```diff
@@ -41,7 +41,7 @@ export const getClusterIdQuery = (cluster: Cluster): NavFilter => {
     // TODO: remove assertion after typing CspFinding as discriminating union
     return { 'cloud.account.id': cluster.meta.cloud!.account.id };
   }
-  return { cluster_id: cluster.meta.assetIdentifierId };
+  return { 'orchestrator.cluster.id': cluster.meta.assetIdentifierId };
```

Comment:
Good call, the integration PR to copy the value is not released, so any deployments with old agent data will not have 'orchestrator.cluster.id'. Are we still sending 'cluster_id' with new agents? If so, maybe we can revert this change until a future release, though this would not guarantee all clients would be updated by then.

Is there some other info we can use to check source event version, and then conditionally use the new field?

Reply:
Updated to use the 'asset_identfier' runtime mapping.

Reply:
Upon further testing, it seems this breaks the dashboard, as the "asset_identifier" runtime mapping does not exist on the "dataView" used to query data for this page. I'm wondering if the best course of action is to leave this as "cluster_id" until everyone updates their integration package (to make use of the copy_from processor).

Reply:
eventually, all our events would have `orchestrator.cluster.id`, so it is a temporary state.

I'd involve @tinnytintin10 and @tehilashn into the discussion here and ask what their opinion in that case.

Tehila and Tin, the question here is, are we ok with having a temporary issue with certain filters redirected from the dashboard. It might take up to 24 hours till it will get fixed.

@mitodrummer another option here is to backport our new processor to previous integrations. That would definitely make sure we have that data prior to 8.10 release

Reply:
I didn't realize you could backport an integration version. I was under the assumption you always need to roll the version forward in order for it to be picked up in kibana. At least from my experience that was what I saw.

Reply:
After chatting with Kfir it sounds like we will roll forward with using 'orchestrator.cluster.id', and do the work to backport the "processor" and "mapping" for the copy_from cluster_id -> orchestrator.cluster.id in the interation package. This could still create a small window (before latest findings are updated) where the field might not be populated, so hopefully that is acceptable.